examples/simple-game-server/Makefile

```diff
@@ -109,17 +105,14 @@ push-windows-image-%:
 	$(MAKE) WINDOWS_DOCKER_PUSH_ARGS=--push build-windows-image-$*
 
 build-windows-image-%: ensure-windows-buildx
-	cd $(root_path) && DOCKER_CLI_EXPERIMENTAL=enabled \
+	cd $(root_path) && \
 		docker buildx build --platform windows/amd64 --builder $(BUILDX_WINDOWS_BUILDER) -f $(project_path)Dockerfile.windows \
 	 	--tag=$(server_tag)-windows_amd64-$* --build-arg WINDOWS_VERSION=$* . $(WINDOWS_DOCKER_PUSH_ARGS)
 
 ensure-windows-buildx:
-	# TODO: Remove `--use` and specify `--builder` for each buildx command once Docker 19.03.13 is available.
-	# `--use` sets the global default buildx context to $(BUILDX_WINDOWS_BUILDER).
-	#
-	# Windows image builds must be directed to a specific buildx context.
-	# The default context does not support building cross platform images.
-	-DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) --use
+# Windows image builds must be directed to a specific buildx context.
+# The default context does not support building cross platform images.
+	-docker buildx create --name=$(BUILDX_WINDOWS_BUILDER)
```

Comment:
```suggestion
	-docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) --use
```

Reply:
I am going to follow what you're suggesting. Just a thought – should we aim for uniformity between the `simple-game-server` and `Xonotic`?
https://github.com/googleforgames/agones/blob/main/examples/xonotic/Makefile#L86-L93

Reply:
Success log: https://gist.github.com/Kalaiselvi84/e6dddbfb6582b563ebb9d5dc2efb43e2